stub/UVFunctions.php

```diff
@@ -0,0 +1,2039 @@
+<?php
+
+/**
+ * create a `new` loop handle.
+ *
+ * @return UVLoop
+ */
+function uv_loop_new()
+{
+}
+
+/**
+ * Returns the initialized default loop.
+ * It may return NULL in case of allocation failure. This function is just a convenient
```

Comment:
Maybe on the C level, but PHP will fatal error before that, Just remove that comment about alloc failure.

Reply:
Will do. 

Also there many other places if viewed, will pop up with a stright line thru the comments that really doesn't to this PHP version. I will go thru and remove those too.